modules/swagger-codegen/src/main/resources/cpprest/model-header.mustache

```diff
@@ -50,11 +51,24 @@ public:
     /// {{description}}
     /// </summary>
     {{^isNotContainer}}{{{datatype}}}& {{getter}}();
-    {{/isNotContainer}}{{#isNotContainer}}{{{datatype}}} {{getter}}() const;
+    {{/isNotContainer}}{{#isNotContainer}}{{{datatype}}} {{getter}}() const;    
     {{/isNotContainer}}{{^required}}bool {{nameInCamelCase}}IsSet() const;
     void unset{{name}}();
     {{/required}}
     void {{setter}}({{{datatype}}} value);
+    
+    {{#isString}}
+    {{{datatype}}}* getNew{{nameInCamelCase}}Instance();
```

Comment:
Why do you need those? Also who frees the memory that you have requested with these functions?

Reply:
The memory is freed exactly as with the prior version in other generated parts of the code. I need these functions to be able to correctly handle the generation of the shared_ptr with an Object. It might seem stupid but this is the only way I found which generates the code correctly without using the prior nullptr (-> segmentation fault) as well as it doesn't touch any other generated code. Every other possible solution broke the build in classes which don't use any Object.